main.py

```diff
@@ -29,47 +29,47 @@
 while True:
     if pool.floatSwitch.get_state() == 0:
```

Comment:
Should these also be constants or is 0 & 1 descriptive enough here?

Reply:
It sure would be a good idea to use constants. Any proposals for the naming? :) I will add the constants here as soon as I get to test the constants you added to Pool-class.

Reply:
Based on the GPIO module should they be `HIGH` & `LOW`? You could also study if the constants from GPIO could be used directly, e.g. `GPIO.OUT`, `GPIO.HIGH` etc.

Reply:
HIGH and LOW are the logic levels of the GPIO-pins. In this case I was trying to abstract the behavior of the float switch from the GPIO... I don't know if it makes any sense! I was thinking about ACTIVE and INACTIVE, or simply ON and OFF.

Reply:
it could be neat also to define those as getter functions: 

```
@property
def ACTIVE(self):
  return GPIO.input(22) == 0

@property
def INACTIVE(self):
  return GPIO.input(22) == 1
...
if pool.floatSwitch.ACTIVE:
...
if pool.floatSwitch.INACTIVE:
...
```

How does that sound?